File: 2.0/python/texts/getTextBroadcast.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.getTextBroadcast(
    id=379506003,
    # return only specific fields
    fields='name,status,labels'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/texts/getTextBroadcast.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getTextBroadcast();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "name,status,labels"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
